docs/sources/flow/reference/components/loki.process.md

```diff
@@ -1362,13 +1362,147 @@ stage.timestamp {
 }
 ```
 
+### stage.geoip block
+
+The `geoip` inner block configures a processing stage that reads an ip address and populates the labelset with geoip fields. Maxmind’s GeoIP2 databse is used for the lookup.
+
+Populated fields for City db:
+
+- geoip_city_name
+- geoip_country_name
+- geoip_continet_name
+- geoip_continent_code
+- geoip_location_latitude
+- geoip_location_longitude
+- geoip_postal_code
+- geoip_timezone
+- geoip_subdivision_name
+- geoip_subdivision_code
+
+Populated fields for ASN (Autonomous System Number) db:
+
+- geoip_autonomous_system_number
+- geoip_autonomous_system_organization
```

Comment:
We typically put detail like this following the arguments table for the block so the table is close as it can be to the header, can you move it down below?

Reply:
The added labels are already well documented in the two examples, so I decided to remove it from the introduction.